icepyx/core/granules.py

```diff
@@ -276,11 +286,11 @@ def place_order(
 
         Parameters
         ----------
-        CMRparams : dictionary
+        CMRparams :
             Dictionary of properly formatted CMR search parameters.
-        reqparams : dictionary
+        reqparams :
             Dictionary of properly formatted parameters required for searching, ordering,
-            or downloading from NSIDC.
+            or downloading from EGI.
```

Comment:
```suggestion
            or downloading from NSIDC (via their EGI system).
```

Nobody outside NSIDC has ever heard of EGI...

Reply:
Personally, I think it's a bit confusing to associate NSIDC and EGI like this (granted, `nsidc` is in the EGI url as it's running on hardware we manage, but I think that's largely irrelevant to the user). All public information about it is under the NASA wiki (https://wiki.earthdata.nasa.gov/display/SDPSDOCS/EGI+Programmatic+Access+Documentation). There are 3 DAACs which are "compatible" with this system: NSIDC, LP, and ASDC. That's just my opinion, I'm fine with whatever you prefer :) Especially since EGI is going away, none of this matters :laughing: I'm hyperfocusing :laughing:

Reply:
> Personally, I think it's a bit confusing to associate NSIDC and EGI like this

Then perhaps keep it at its original "downloading from NSIDC"? I think it would be more confusing to users if they somehow think they're not downloading their data from NSIDC because we're naming the specific endpoint being hit (and underlying tool being used). After the update we can try to explain this clearly in the tutorials (if it's not already).

Reply:
I do think your suggestion is an improvement over the original, because at least then there's a way to Google for the term!